### PR TITLE
🍱 [server] Update manifest and add logos

### PR DESCRIPTION
**Pull request by rhinodavid**
Read more about manifests at
https://developers.google.com/web/fundamentals/web-app-manifest.

### CODE REVIEW

**Comment by rhinodavid** [comment]
logo-512
 Logo looks like dis.

File: packages/upswyng-server/static/manifest.json [hunk 1]
```diff
@@ -1,8 +1,20 @@
 {
   "background_color": "#ffffff",
-  "theme_color": "#333333",
-  "name": "TODO",
-  "short_name": "TODO",
+  "theme_color": "#f15b29",
```
Comment (jacobvenable) [inline packages/upswyng-server/static/manifest.json]: uhm, this isn't [our orange](https://github.com/CodeForBoulder/upswyng/blob/master/packages/upswyng-web/src/App.styles.ts#L19). This looks [waaaaayyyyy different](https://www.google.com/search?q=%23F05A28&oq=%23F05A28&aqs=chrome..69i57j0.1257j0j7&sourceid=chrome&ie=UTF-8) 😏

Comment (rhinodavid) [reply packages/upswyng-server/static/manifest.json]: Ummm you sure?

https://github.com/CodeForBoulder/upswyng/blob/master/packages/upswyng-server/src/style/main.scss#L1

Screen Shot 2020-01-27 at 9 36 39 PM

Comment (rhinodavid) [reply packages/upswyng-server/static/manifest.json]: Screen Shot 2020-01-27 at 9 37 25 PM

Comment (jacobvenable) [reply packages/upswyng-server/static/manifest.json]: Is that derived from the SVG or PNG? Either way, I really was just joking :P

Comment (rhinodavid) [reply packages/upswyng-server/static/manifest.json]: It's from some Illustrator file in one of Mike's Google Drive folders. 🤷🏾‍♀️ I legit thought you had like crazy color perception or something.

**Pull request merged**